composer.json

```diff
@@ -34,11 +34,11 @@
     ],
     "require-dev": {
         "doctrine/coding-standard": "^11.1.0",
-        "phpstan/phpstan": "^1.9.16",
-        "phpstan/phpstan-phpunit": "^1.3.3",
-        "phpunit/phpunit": "10.0.5",
+        "phpstan/phpstan": "^1.9.17",
+        "phpstan/phpstan-phpunit": "^1.3.4",
+        "phpunit/phpunit": "^10.0.7",
```

Comment:
If I follow your reasoning, this patch should be red also on the pre-existing PHPUnit version?

Reply:
Yep, even in the last PHPUnit 9.6.3 build you had more than a thousand mutants escaped under the timeout carped:

https://github.com/Roave/BetterReflection/actions/runs/4092019766/jobs/7056471771#step:5:117

```
3676 mutations were generated:
    2567 mutants were killed
       0 mutants were configured to be ignored
       0 mutants were not covered by tests
       0 covered mutants were not detected
       5 errors were encountered
       0 syntax errors were encountered
       3 time outs were encountered
    1101 mutants required more time than configured
```